packages/mds-orm/entities/audit-entity.ts

```diff
@@ -17,7 +18,7 @@ export interface AuditPersistenceModel extends RecordedPersistenceModel {
 }
 
 @Entity('audits')
-export class AuditEntity extends RecordedEntity implements AuditPersistenceModel {
+export class AuditEntity extends RecordedEntity implements AuditPersistenceModel, IdentityPersistenceModel {
```

Comment:
`AuditPersistenceModel` extends `IdentityPersistenceModel` ... You should be able to access `id` without `implements IdentityPersistenceModel`